### DISCUSSION TITLE
Meta blocks

### DISCUSSION THREAD

**Discussion by evan-forbes** [celestiaorg/celestia-app#2058] [Ideas]
We've casually discussed the idea of producing blocks as fast as possible, and then combining the individual smaller blocks into a larger "meta block" that gets sampled by light clients, so I'm opening the discussion to write down our collective thoughts.

This is also a crude mechansim for "multiplicity". For example if we have 1 second blocks and a 15 second meta block that includes all the block data for the past 15 blocks, then we apply some arbitary function to the past 15 blocks that deterministically orders the blobs included by up to 15 different block producers.

**Comment by musalbas** [comment]
Sounds potentially similar to a shared mempool protocol like [Narwhal](https://arxiv.org/abs/2105.11827)
